inc/sitemaps/class-sitemaps.php

```diff
@@ -230,6 +230,12 @@ public function redirect( $query ) {
 			return;
 		}
 
+		$type = get_query_var( 'sitemap' );
```

Comment:
It will prevent generating Yoast main (or news) stylesheet (if _sitemap_ is empty) . I think that we should revert this change.

Reply:
Sorry, not clear what you mean with "I think that we should revert this change."

Reply:
I just explained more details in comment. It's an issue because the ordering of checking is changed (_sitemap_ vs _xsl_). It will break news stylesheet.